assets/stylesheets/common/common.scss

```diff
@@ -504,6 +503,8 @@ $float-height: 530px;
         }
 
         .emoji {
+          height: 15px;
```

Comment:
Emojis are not all the same aspect ratio. You really shouldn't set a height on them. I tried this and it doesn't look right when you have enough emojis IMO. I would have height set or width set and have the other unset.

Reply:
Okay, changed width to `auto`, but are there really any non-square emoji?